models/demos/wormhole/squeezebert/tt/ttnn_functional_squeezebert.py

```diff
@@ -0,0 +1,611 @@
+        memory_config=ttnn.L1_MEMORY_CONFIG,
+    )
+    ttnn.deallocate(embeddings)
+
+    encoder_output = squeezebert_encoder(
+        config=config,
+        hidden_states=encoder_input,
+        attention_mask=attention_mask,
+        state_dict=state_dict,
+        base_addr=f"{base_addr}encoder.",
+        parameters=parameters.encoder,
+        device=device,
+        reader_patterns_cache=reader_patterns_cache,
+        mesh_mapper=mesh_mapper,
+        mesh_composer=mesh_composer,
+    )
+    ttnn.deallocate(encoder_input)
+
+    return encoder_output
+
+
+def squeezebert_for_question_answering(
+    config,
+    input_ids,
+    token_type_ids,
+    position_ids,
+    attention_mask,
+    *,
+    state_dict,
+    base_addr,
+    parameters,
+    device,
+    reader_patterns_cache,
+    name="transformer",
+    mesh_mapper=None,
+    mesh_composer=None,
+):
+    # print(f"parmes: {parameters.keys()}")
+    squeezebert_output = squeezebert(
+        config,
+        input_ids,
+        token_type_ids,
+        position_ids,
+        attention_mask,
+        state_dict,
+        base_addr,
+        parameters=parameters.transformer,
+        device=device,
+        reader_patterns_cache=reader_patterns_cache,
+        mesh_mapper=mesh_mapper,
+        mesh_composer=mesh_composer,
+    )
+
+    qa_outputs = ttnn.linear(
+        squeezebert_output,
+        parameters.qa_outputs.weight,
+        bias=parameters.qa_outputs.bias,
+        memory_config=ttnn.L1_MEMORY_CONFIG,
+    )
+
+    return qa_outputs
+
+
+def preprocess_inputs(
+    input_ids,
+    token_type_ids,
+    position_ids,
+    attention_mask,
+    device,
+    mesh_mapper=None,
+):
+    import torch
+
+    batch_size, _ = input_ids.shape
+
+    input_ids = ttnn.from_torch(
+        input_ids,
+        dtype=ttnn.uint32,
+        mesh_mapper=mesh_mapper,
+        device=device,
+        memory_config=ttnn.L1_MEMORY_CONFIG,
+    )
+    token_type_ids = ttnn.from_torch(
+        token_type_ids,
+        dtype=ttnn.uint32,
+        mesh_mapper=mesh_mapper,
+        device=device,
+        memory_config=ttnn.L1_MEMORY_CONFIG,
+    )
+    position_ids = ttnn.from_torch(
+        position_ids, dtype=ttnn.uint32, mesh_mapper=mesh_mapper, device=device, memory_config=ttnn.L1_MEMORY_CONFIG
+    )
+
+    if attention_mask is not None:
+        attention_mask = get_extended_attention_mask(attention_mask, input_ids.shape, torch.float32)
+        attention_mask = attention_mask.expand((batch_size, -1, -1, -1))
+        attention_mask = torch.clamp(attention_mask, min=-100000)
+        attention_mask = ttnn.from_torch(
+            attention_mask,
+            dtype=ttnn.bfloat16,
+            layout=ttnn.TILE_LAYOUT,
+            mesh_mapper=mesh_mapper,
+            device=device,
+            memory_config=ttnn.L1_MEMORY_CONFIG,
+        )
+
+    return input_ids, token_type_ids, position_ids, attention_mask
+
+
+# def preprocess_weight(weight, mesh_mapper, mesh_device):
```

Comment:
remove commented code

Reply:
I've removed the commented code. 
Thank you.